deploy/infra/lib/helpers/base-lambda.ts

```diff
@@ -0,0 +1,28 @@
+import * as cdk from "aws-cdk-lib";
+import * as lambda from "aws-cdk-lib/aws-lambda";
+import { PolicyStatement } from "aws-cdk-lib/aws-iam";
+
+export interface BaseLambdaFunProps {
+  functionProps: lambda.FunctionProps;
+  vpcConfig: any;
+}
+
+export class BaseLambdaFunction extends lambda.Function {
+  constructor(scope: any, id: string, props: BaseLambdaFunProps) {
+    super(scope, id, props.functionProps);
+    this.addToRolePolicy(
```

Comment:
What about attach the default policy from AWS AWSLambdaVPCAccessExecutionRole. https://docs.aws.amazon.com/aws-managed-policy/latest/reference/AWSLambdaVPCAccessExecutionRole.html 

it is almost the same